src/test/java/org/folio/sample/integration/api/books/GetLeapYearBookTest.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.sample.integration.api.books;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+
+import io.restassured.common.mapper.TypeRef;
+import io.restassured.response.Response;
+import java.time.LocalDate;
+import java.util.List;
+import org.folio.sample.domain.dto.BookDTO;
+import org.folio.sample.domain.dto.BookForCreationDTO;
+import org.folio.sample.integration.AbstractBaseApiTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+public class GetLeapYearBookTest extends AbstractBaseApiTest {
```

Comment:
See [this](https://sonarcloud.io/project/issues?resolved=false&pullRequest=6&id=ualibweb_folio-training-backend&open=AY7D-Xktv75itiuB6xBx) code smell.

A code smell is a potential bug or issue that is not breaking functionality now, but can potentially in the future or it is bad practice, etc. It is found by Sonarcloud which is an automated tool that runs on this pull request.